docs/docs/tutorial_landsat_ndvi.md

```diff
@@ -0,0 +1,701 @@
+Landsat NDVI computation
+========================
+
+Actinia provides several API calls to compute satellite specific
+parameters:
+
+ <https://actinia.mundialis.de/api_docs/#tag-Satellite-Image-Algorithms>
+
+ We will use the Unix shell and curl to access the REST API. First open a shell of choice (we use bash here) and setup the login information, the  IP address and the port on which the actinia service is running, so you can simply change the IP and port if your server uses a different
+ address:
+
+ ```bash
+ export ACTINIA_URL=https://actinia.mundialis.de/latest
+ export AUTH='-u demouser:gu3st!pa55w0rd'
+ # other user credentials can be provided in the same way
+ ```
+
+The NDVI is an important parameter that is derived from multi-spectral
+satellite images. The following asynchronous API call computes the NDVI
+of the Landsat8 scene **LC80440342016259LGN00** with TOAR top of
+atmosphere correction. It is designed to hide the complexity of Landsat
+scene downloading, reprojection, atmospheric correction, statistical
+analysis and preview rendering in a single call using a self describing
+url.
+
+
+```bash
+ curl ${AUTH} -X POST -i "${ACTINIA_URL}/landsat_process/LC80440342016259LGN00/TOAR/NDVI"
```

Comment:
Computing the NDVI of the Landsat8 scene LC80440342016259LGN00 with TOAR top of atmosphere correction results in following error:

16:04 $ curl ${AUTH} -X POST "${ACTINIA_URL}/landsat_process/LC80440342016259LGN00/TOAR/NDVI"

...

16:19 $ curl ${AUTH} -X GET "https://actinia.mundialis.de/api/v1/resources/demouser/resource_id-858241cf-08cc-4c54-abe9-db08c3f4baac" | jq
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  1595  100  1595    0     0   4310      0 --:--:-- --:--:-- --:--:--  4310
{
  "accept_datetime": "2021-05-28 14:13:02.701484",
  "accept_timestamp": 1622211182.7014792,
  "api_info": {
    "endpoint": "asyncephemerallandsatprocessingresource",
    "method": "POST",
    "path": "/api/v1/landsat_process/LC80440342016259LGN00/TOAR/NDVI",
    "request_url": "http://actinia.mundialis.de/api/v1/landsat_process/LC80440342016259LGN00/TOAR/NDVI"
  },
  "datetime": "2021-05-28 14:13:02.955742",
  "exception": {
    "message": "'LC80440342016259LGN00'",
    "traceback": [
      "  File \"/usr/lib/python3.8/site-packages/actinia_core/resources/ephemeral_processing.py\", line 1651, in run\n    self._execute()\n",
      "  File \"/usr/lib/python3.8/site-packages/actinia_satellite_plugin/ephemeral_landsat_ndvi_processor.py\", line 1002, in _execute\n    download_pl, file_infos = process_lib.get_download_process_list()\n",
      "  File \"/usr/lib/python3.8/site-packages/actinia_core/resources/common/geodata_download_importer.py\", line 156, in get_download_process_list\n    self._setup()\n",
      "  File \"/usr/lib/python3.8/site-packages/actinia_core/resources/common/landsat_processing_library.py\", line 189, in _setup\n    for suffix in SCENE_SUFFIXES[self.landsat_sensor_id]:\n"
    ],
    "type": "<class 'KeyError'>"
  },
  "http_code": 400,
  "message": "'LC80440342016259LGN00'",
  "process_chain_list": [],
  "process_log": [],
  "progress": {
    "num_of_steps": 0,
    "step": 0
  },
  "resource_id": "resource_id-858241cf-08cc-4c54-abe9-db08c3f4baac",
  "status": "error",
  "time_delta": 0.2543332576751709,
  "timestamp": 1622211182.9556062,
  "urls": {
    "resources": [],
    "status": "https://actinia.mundialis.de/api/v1/resources/demouser/resource_id-858241cf-08cc-4c54-abe9-db08c3f4baac"
  },
  "user_id": "demouser"
}